components/button/disabled-button.md

```diff
@@ -12,6 +12,8 @@ position: 5
 
 Sometimes specific buttons in an application must be temporarily disabled. To control the enabled state of the component, use the `Enabled` Boolean attribute.
 
+**Note:** The component's `OnClick` event handler does not validate its `Enabled` state before actually firing the event. Setting `Enabled` to false only affects the rendering of the HTML button element by setting its `disabled` attribute. If this is removed via DOM manipulation the button can still be clicked and the event will be fired.
```

Comment:
We avoid gerund and passive voice (https://docs.telerik.com/style-guide/introduction)
Switched to a styled note block.

```suggestion
> `Enabled="false"` renders a `disabled` attribute on the `<button>` element. A disabled Button will fire its `OnClick` handler if the user removes the `disabled` attribute via the browser's developer console. This behavior is consistent with standard buttons.
```